src/components/Layout/Contact.test.tsx

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from "@testing-library/react";
+import Contact from "./Contact";
+
+test("renders Contact component", () => {
+  render(<Contact />);
+  const contactElement = screen.getByText(/contact/i);
+  expect(contactElement).toBeInTheDocument();
+});
```

Comment:
This the basic structure of a React component test: render -> find element -> verify something about element.